modules/dashboard/php/NDB_Form_dashboard.class.inc

```diff
@@ -293,7 +293,7 @@ class NDB_Form_Dashboard extends NDB_Form
                         "SELECT Test_name from flag WHERE CommentID=:cid",
                         array('cid' => $bvl_feedback[$i]['CommentID'])
                     );
-                    if ($instrument !== null) {
+                    if (!empty($instrument)) {
```

Comment:
@driusan what if someone calls their instrument "0" that's going to be problematic no ?

Reply:
Yes.

Reply:
ok